addons/mail/static/src/models/file_uploader.js

```diff
@@ -148,6 +148,9 @@ registerModel({
                     }
                 }
             }
+            for (const i in attachments) {
+                (composer || thread).messaging.models['Attachment'].insert(attachments[i]);
+            }
             if (activity && activity.exists()) {
                 await activity.markAsDone({ attachments });
             }
```

Comment:
`attachments` were `Attachment` records, but they are now data. This means `activity.markAsDone({ attachments })` passes the wrong parameter now.